docs/Operators.md

```diff
@@ -10237,6 +10238,91 @@ This version of the operator has been available since version 1 of the default O
 </dl>
 
 
+### <a name="Range"></a><a name="range">**Range**</a>
+
+  Generate a tensor containing a sequence of numbers that begin at `start` and extends by increments of `delta` 
```

Comment:
Can you add a pseudocode or a simple example in the doc?

Reply:
Added pseudocode and some examples.